app/admin/properties.rb

```diff
@@ -0,0 +1,61 @@
+ActiveAdmin.register Property do
+
+  # See permitted parameters documentation:
+  # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+
+  permit_params do
+    permitted = [:name, :description, :address, :city, :price, :user_id, :is_paid, :buyer_id, images: [] ]
+    permitted << :other if params[:action] == 'create' && current_user.admin?
```

Comment:
Just for the knowledge, Can you please explain me know, That which kind of work we can achieve using this code?

Reply:
- done